src/libcommon/keychainmanager/keychainmanager.cpp

```diff
@@ -76,7 +76,7 @@ bool KeyChainManager::writeToken(const std::string &keychainKey, const std::stri
     keychain::setPassword(PACKAGE, SERVICE, keychainKey, rawData, error);
     if (error) {
         LOG_DEBUG(KDC::Log::instance()->getLogger(),
-                  "Failed to save authentification info to keychain: " << error.code << " - " << error.message.c_str());
```

Comment:
Suggestion: use `KDC::Utility::formatStdError` in combination with `LOGW_DEBUG`.

Reply:
Why? There is no UTF16 string in this message? This wil generate 2 unneccessary string conversion

Reply:
This is more about using a common format for logging errors through the code base, and also typing a bit less. 
(An helper could be added to return `std::string` when `std::wstring` is not required). It's up to you; just a suggestion.

Reply:
Discussed together: this is not a std::error but a keychain::error